src/smc-webapp/editor_chat.cjsx

```diff
@@ -291,6 +301,15 @@ class ChatActions extends Actions
         w.document.write('</body></html>')
 
         w.addEventListener "unload", () =>
+            video = (@store.get('video')?.toJS()) ? {}
```

Comment:
I was concerned about this conversion from Immutable but looks like `@save_shared_video_info` expects an object, so in this context it's fine I guess.